src/main/java/ui/windows/StatsRendererData.java

```diff
@@ -0,0 +1,33 @@
+package ui.windows;
+
+import javax.swing.*;
+
+public class StatsRendererData {
+    private String name;
+    private int gamesPlayed;
+    private int gamesWon;
+    private int longestWinStreak;
+
+    public StatsRendererData(String name, int gamesPlayed, int gamesWon, int longestWinStreak) {
```

Comment:
This is a pretty long parameter list with parameters being related to each other. You could package these together into a single object, or replace it with a method call. I believe that since this data is coming from the Data Access Interface, these should be packaged in to an Output Data object in the use case layer, and perhaps the attributes in this class can be set by getter methods (make a StatsOutput object or something similar, pass it in to there, and unpackage its attributes with getter methods).